src/greedy.jl

```diff
@@ -116,7 +116,49 @@ end
 ##===================================================##
 
 """
-    greedy(instance::Instance)
+    filter_on_max_criterion!(candidates::Array{Int64,1}, criterion)
+
+Takes a set of candidates and removes all elements with bad criterion.
+"""
+function filter_on_max_criterion(candidates::Array{Int64,1}, criterion) # TODO: criterion is Int or Float
```

Comment:
`criterion::Union{Float64, Int}` see [this](https://docs.julialang.org/en/v1/manual/types/#Type-Unions-1) for more info